scripts/docker/run.sh

```diff
@@ -59,30 +65,32 @@ touch $XAUTH
 xauth nlist $DISPLAY | sed -e 's/^..../ffff/' | xauth -f $XAUTH nmerge -
 
 if [ "$os" = "xenial" ]; then
-docker run -it --rm --name astrobee \
-        --volume=$XSOCK:$XSOCK:rw \
-        --volume=$XAUTH:$XAUTH:rw \
-        --env="XAUTHORITY=${XAUTH}" \
-        --env="DISPLAY" \
-        --gpus all \
-      astrobee/astrobee:latest-ubuntu16.04 \
-    /astrobee_init.sh roslaunch astrobee sim.launch $args
+localtag=astrobee/astrobee:latest-ubuntu16.04
+remotetag=ghcr.io/nasa/astrobee:latest-ubuntu16.04
 elif [ "$os" = "bionic" ]; then
-docker run -it --rm --name astrobee \
-        --volume=$XSOCK:$XSOCK:rw \
-        --volume=$XAUTH:$XAUTH:rw \
-        --env="XAUTHORITY=${XAUTH}" \
-        --env="DISPLAY" \
-        --gpus all \
-      astrobee/astrobee:latest-ubuntu18.04 \
-    /astrobee_init.sh roslaunch astrobee sim.launch $args
+localtag=astrobee/astrobee:latest-ubuntu18.04
+remotetag=ghcr.io/nasa/astrobee:latest-ubuntu18.04
 elif [ "$os" = "focal" ]; then
+localtag=astrobee/astrobee:latest-ubuntu20.04
+remotetag=ghcr.io/nasa/astrobee:latest-ubuntu20.04
+fi
+
+# check if docker tag exists
```

Comment:
it's easier to just define tag as a default in the start and overwrite it if there are arguments defined by the user (-r)

Reply:
Yep, fixed!